packages/web-runtime/src/components/SidebarNav/SidebarNav.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div id="web-nav-sidebar" :class="sidebarClasses">
+    <oc-button
+      variation="inverse"
```

Comment:
a lot of hardcoded inverse variations, (also on other places). Could be problematic in case of using a bright theme